compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -12721,17 +12721,17 @@ TR::Node *ificmpeqSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
 
    bitwiseToLogical(node, block, s);
 
-   if (firstChild->getOpCode().isBooleanCompare() && !firstChild->getOpCode().isLongCompare() &&
+   if (firstChild->getOpCode().isBooleanCompare() &&
        (secondChild->getOpCode().isLoadConst()) &&
-       ((secondChild->getInt() == 0) || (secondChild->getInt() == 1)) &&
+       ((secondChild->getLongInt() == 0) || (secondChild->getLongInt() == 1)) &&
```

Comment:
secondChild is the second child of ificmpeq and so must be an iconst to have a type correct tree; given this, why is getInt not the right api to call ?